apollo-router/tests/snapshots/lifecycle_tests__cli_config_experimental.snap

```diff
@@ -1,6 +1,6 @@
 ---
 source: apollo-router/tests/lifecycle_tests.rs
-expression: "command_output(Command::new(IntegrationTest::router_location()).arg(\"config\").arg(\"experimental\")).await"
+expression: "command_output(Command::new(IntegrationTest::router_location()).arg(\"config\").arg(\"experimental\").env(\"RUST_BACKTRACE\",\n            \"\")).await"
```

Comment:
Just FYI that this line is here. (Didn't see it called out in PR desc, so just flagging.)

Reply:
I believe it is due to a code change made in an unrelated PR since this snapshot file was last created. (Snapshot tests still succeed when only metadata changes.)